aldryn_search/helpers.py

```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*-
+from cms.plugin_rendering import PluginContext
+import django
```

Comment:
I think this is correct. **d**jango comes after **c**ms (both are external libraries).